source/game.person.base.bmx

```diff
@@ -646,12 +642,26 @@ Type TPersonBase Extends TGameObject
 	End Method
 
 
+	Method GetNickName:String()
+		If nickName = "" Then Return firstName
+		Return nickName
+	End Method
+
+
 	Method GetFullName:String()
```

Comment:
Ich glaube, in der Expressionauswertung gesehen zu haben, dass fullname dort selbst zusammengebaut wird - sollte das nicht an diese Methoden delegiert werden?

Reply:
Dein "ich glaube" bezieht sich womoeglich auf den "PersonGenerator"...
```
		case "fullname"   Return New SToken( TK_TEXT, GetPersonGenerator().GetFirstName(country, gender) + " " + GetPersonGenerator().GetLastName(country, gender), params.GetToken(0) )
```
Ich baue dort mal eine GetFullName(country,gender) mit ein - da in dem Fall ja beides "gleich" ist, braucht es dort keinen Separator.